features.bzl

```diff
@@ -8,6 +8,9 @@ _cc = struct(
     # on find_cpp_toolchain are available (#17308).
     # Note: While the target and parameter are available in 6.1.0, they only take effect in Bazel 7.
     find_cpp_toolchain_has_mandatory_param = ge("6.1.0"),
+    # Note: In Bazel 6.3 the `grep_includes` parameter is optional and a no-op in the cc_common API
+    # In future Bazel versions it will be removed altogether.
+    remove_grep_includes_from_api = ge("6.3.0"),
```

Comment:
Probably should rename it as `grep_includes_is_no_op`? Since it's not actually removed.

Reply:
Maybe `grep_includes_is_optional` since that's what most API users will care about?

Reply:
I think both of those are fine, but in a future Bazel release the `grep_includes` parameter will not exist at all, so I guess we'd need to add another `grep_includes_is_removed`?

Reply:
Renamed to `grep_includes_is_optional`.

Reply:
Yes, if users actually want to specify this attribute, we may need this feature in the future - we can always add it if there is demand for it.